packages/venia-ui/lib/components/CartPage/ProductListing/quantity.js

```diff
@@ -11,6 +11,9 @@ import TextInput from '../../TextInput';
 import { Message } from '../../Field';
 import defaultClasses from './quantity.module.css';
 
+/**
+ * @deprecated - use components/QuantityStepper instead
```

Comment:
This is on the right track, but we can additionally get rid of this duplicate code now. We had a bunch of this when we migrated graphql operations from venia-ui to peregrine, and just in case a third-party dev was using them, we used this re-export and alias pattern.

https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.gql.js

Would expect something like `export { default as QuantityFields } from '../../QuantityStepper'`. This probably means you can also cleanup style and test files, since those are redundant as well.